src/urbit/app/chess.hoon

```diff
@@ -114,158 +156,241 @@
                 result=~
                 moves=~
             ==
-          :-  :~  :*  %pass  /player/(scot %da game-id)
-                      %agent  [who.action %chess]
-                      %watch  /game/(scot %da game-id)/moves
-                  ==
-                  :*  %give  %fact  ~[/active-games]
-                      %chess-game  !>(new-game)
-                  ==
+          ::  return [(list card) _this]
```

Comment:
Which is `(quip card _this)`, which is the return type for `on-poke` and most agent arms